corehq/apps/registration/static/registration/js/login.js

```diff
@@ -6,4 +6,16 @@ $(function () {
     new Blazy({
         container: 'body',
     });
+
+    hqImport("analytix/js/hubspot").then(function () {  // because hubspot loads after kissmetrics
```

Comment:
Not quite following this. Is this to prevent the kiss events from firing if hubspot hasn't yet fully loaded?

Reply:
i guess this made sense when it was in `register_new_user.js` but now it's not. updated